src/containerapp/azext_containerapp/custom.py

```diff
@@ -152,6 +156,35 @@ def create_deserializer():
     return Deserializer(deserializer)
 
 
+def list_all_services(cmd, environment_name, resource_group_name):
+    services = list_containerapp(cmd, resource_group_name=resource_group_name, managed_env=environment_name)
+    services = [c for c in services if c["properties"]["configuration"]["service"] is not None and
```

Comment:
use safe_get to get property to avoid exception

Reply:
Fixed